### DISCUSSION TITLE
Issues when working on projects with a rust-toolchain.toml

### DISCUSSION THREAD

**Discussion by JoshFried** [mrcjkb/rustaceanvim#385] [Q&A]
When I work on a project that contains a rust-toolchain.toml my neovim intsance freezes. 

I was wondering if anyone knew the way to properly configure this plugin to handle this.

**Comment by mrcjkb** [comment]
Hey 👋

This plugin doesn't have any explicit behaviour implemented for projects with a rust-toolchain.toml.
So it's likely a rustup issue.
